sos/report/plugins/ubuntu.py

```diff
@@ -22,3 +22,12 @@ def setup(self):
             "hwe-support-status --verbose",
             "ubuntu-advantage status"
         ])
+
+        if not self.get_option("all_logs"):
+            self.add_copy_spec([
+                "/var/log/ubuntu-advantage.log",
+                "/var/log/ubuntu-advantage.log.1",
+                "/var/log/ubuntu-advantage.log.2*",
```

Comment:
Is this trailing glob intentional? It seems odd to me if so, since you'd lose the intermediate rotated logs between 2 and 20.

Reply:
It's intentional to grab the file whether it's `/var/log/ubuntu-advantage.log.2` or `/var/log/ubuntu-advantage.log.2.gz` in this case.
I actually left it like this to somewhat mimic what it did in [the logs plugin (with syslog)](https://github.com/sosreport/sos/blob/8745eb0ac0baed51f89ff9afaf36220d592a119f/sos/report/plugins/logs.py#L82).
The thought about 2 vs 20 (etc) did cross my mind, though, yes. I can make that a better match to limit to "2 or 2.*" if that would be preferred?

Reply:
That's how `ubuntu-advantage-tools` logrotate logs on my Ubuntu machine:
```
/var/log/ubuntu-advantage.log    
/var/log/ubuntu-advantage.log.1  
=> /var/log/ubuntu-advantage.log.2.gz 
/var/log/ubuntu-advantage.log.3.gz
/var/log/ubuntu-advantage.log.4.gz
```
logrotate conf:
```
/var/log/ubuntu-advantage.log {
    rotate 6
    monthly
    compress
    delaycompress
    missingok
    notifempty
}
```
With current configuration, it will never exceed 6.

Reply:
Ah, ok - makes sense, thanks for the context.